src/helpers/PolarGridHelper.js

```diff
@@ -16,7 +16,7 @@ class PolarGridHelper extends LineSegments {
 
 		// create the radials
 
-		for ( let i = 0; i <= radials; i ++ ) {
+		for ( let i = 0; i < radials; i ++ ) {
 
 			const v = ( i / radials ) * ( Math.PI * 2 );
```

Comment:
Now the maximum value of v would be less than 2*Pi, which will result in radials not making full circle. Is this what we want?

Reply:
We don't need v to go completely around the circle because that will create two lines at 0 degrees instead of only one.